src/Models/Daemon.php

```diff
@@ -0,0 +1,146 @@
+<?php
+
+namespace Cyberfusion\ClusterApi\Models;
+
+use Cyberfusion\ClusterApi\Support\Arr;
+use Cyberfusion\ClusterApi\Support\Validator;
+
+class Daemon extends ClusterModel
+{
+    private string $name;
+    private string $command;
+    private int $unixUserId;
+    private array $nodeIds;
+    private ?int $id;
+    private int $clusterId;
+
+    public function getName(): string
+    {
+        return $this->name;
+    }
+
+    public function setName(string $name): self
+    {
+        Validator::value($name)
+        ->pattern('^[a-zA-Z0-9]')
+        ->validate();
+
+        $this->name = $name;
+
+        return $this;
+    }
+
+    public function getCommand(): string{
+        return $this->command;
+    }
+
+    public function setCommand(string $command): self
+    {
+        Validator::value($command)
+        ->validate();
```

Comment:
Styling

Reply:
Still incorrect, but that's hard to notice because the workflow wasn't running. It is now.

Reply:
Calls aren't indented. For comparison: https://github.com/CyberfusionIO/cyberfusion-cluster-api-client/blob/a97b757e2064d2979370dcdfc4ef5ab508dc82b7/src/Models/Token.php#L22

Reply:
https://github.com/CyberfusionIO/cyberfusion-cluster-api-client/actions/runs/9014824015/job/24769215569?pr=109